openapi_python_client/__init__.py

```diff
@@ -239,15 +240,30 @@ def _build_api(self) -> None:
         client_path.write_text(client_template.render(), encoding=self.file_encoding)
 
         # Generate endpoints
+        endpoint_collections_by_tag = self.openapi.endpoint_collections_by_tag.items()
         api_dir = self.package_dir / "api"
         api_dir.mkdir()
-        api_init = api_dir / "__init__.py"
-        api_init.write_text('""" Contains methods for accessing the API """', encoding=self.file_encoding)
+        api_init_path = api_dir / "__init__.py"
+        api_init_template = self.env.get_template("api_init.py.jinja")
+        api_init_path.write_text(
+            api_init_template.render(
+                package_name=self.package_name,
+                endpoint_collections_by_tag=endpoint_collections_by_tag,
+            ),
+            encoding=self.file_encoding,
+        )
 
         endpoint_template = self.env.get_template("endpoint_module.py.jinja")
-        for tag, collection in self.openapi.endpoint_collections_by_tag.items():
+        for tag, collection in endpoint_collections_by_tag:
             tag_dir = api_dir / tag
             tag_dir.mkdir()
+
+            endpoint_init_path = tag_dir / "__init__.py"
+            endpoint_init_template = self.env.get_template("endpoint_init.py.jinja")
+            endpoint_init_path.write_text(
+                endpoint_init_template.render(package_name=self.package_name, tag=tag, endpoints=collection.endpoints),
```

Comment:
Agreed resolved in https://github.com/triaxtec/openapi-python-client/pull/403/commits/cdd304c1a92b208d04541aa1253ebdae39a8a45b

Regarding the `package_name` I do have specific needs for it, so definitly prefer to keep it indeed 😁
I've made a change, package metadata are now wildly available in all templates https://github.com/triaxtec/openapi-python-client/pull/403/commits/77acd253bb60909c31a67db516bc11e9154be68e

Reply:
Please review other template rendering in this file where `package_name` is passed.
Also I see some renders with `description` and `version` instead of `package_description` and `package_version` so that could be unified.

Reply:
Indeed, normalized on https://github.com/triaxtec/openapi-python-client/pull/403/commits/d1120f226da02a811ae10e4d95590218745c2fd8

I've also wildly spread the project metadata vars to all template

Reply:
@dbanty what do you thinks about it? We are including a breaking change here by renaming `{version, description}` to `{pacakge_version, package_description}`.

Imo, `package_X` kind of make more sense

Reply:
The template API is not stable, so I don't care too much about breaking changes to it 😅. So I'm good with adding some clarity / consistency.